kuma/javascript/src/header/login.jsx

```diff
@@ -47,6 +48,11 @@ export default function Login(): React.Node {
                     right={true}
                     hideArrow={true}
                 >
+                    <li>
+                        <a href={contributionsLink}>
+                            {gettext('Contributions')}
```

Comment:
Would it be advantageous to call this "Your contributions" instead? After all, we say "Your profile".

Reply:
Well, this is per design and also, `View profile` and `Edit profile` will be replaced by `Account Settings` so I reckon just `Contributions` is fine.